### ISSUE TITLE
False positive: let_var_whitespace comments with properties

### ISSUE THREAD

**Issue by MrCOBA** [realm/SwiftLint#4871]
### Describe the bug

Hi! Found a new false positive case of let_var_whitespace. The problem is reproduced on SwiftLint version `0.51.0`.
There is error, when the comment style is as follows `/**/`:
```Swift
static var test: String { /* Some Field */
    return "Test!"
}
```
But it's all good if the style is as follows `//`
```Swift
static var test: String { // Some Field
    return "Test!"
}
```
### Environment

SwiftLint version `0.51.0`
Installation method used swiftlint binary
Paste your configuration file:
```
only_rules:
  - let_var_whitespace
```
Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)? No
Which Xcode version are you using? 14.2
Do you have a sample that shows the issue?
```Swift
static var test: String { /* Some Field */
    return "Test!"
}
```
